How Fonts Should Degrade (#30159)
freeCodeCamp/freeCodeCamp: guide/english/css/fonts/index.md

  font: italic small-caps 800 20px/1.5 Arial;
}
```

> It is also possbible to import a new font:

Reviewer: ```suggestion
It is also possible to import a new font:
```

```html
<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css"> 
```

> In this was we import the `lobster`

Reviewer: ```suggestion
Here, the `Lobster` font has been imported
```

> the in the css we can use such font:

Reviewer: ```suggestion
In the page style, it can be used in the following form:
```

```css
  h2 {
    font-family: Lobster, monospace;
  }
```

> In this way the font `Lobster` will be the first choice, but if it is not avaiable the `monospace` will replace it.

Reviewer: ```suggestion
In this way the font `Lobster` will be the first choice, but if it is not available the `monospace` font will replace it.
```
